.github/workflows/build.yml

```diff
@@ -21,26 +21,26 @@ jobs:
     outputs:
       go-version: ${{ steps.get-go-version.outputs.go-version }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Determine Go version
         id: get-go-version
         # We use .go-version as our source of truth for current Go
         # version, because "goenv" can react to it automatically.
         run: |
           echo "Building with Go $(cat .go-version)"
-          echo "::set-output name=go-version::$(cat .go-version)"
+          echo "go-version=$(cat .go-version)" >> $GITHUB_OUTPUT
```

Comment:
where is $GITHUB_OUTPUT being used?

Reply:
It is a magic environment variable it github actions that assigns the output of a command so that you can use it in another action. The `::set-output` way of doing outputs is deprecated.